pkg/rancher-components/src/components/Form/ToggleSwitch/ToggleSwitch.vue

```diff
@@ -41,7 +41,7 @@ export default Vue.extend({
   },
 
   methods: {
-    toggle(neu: boolean | string | number) {
+    toggle(neu: boolean | 'true' | 'false' | undefined | null) {
```

Comment:
If this type could be the same as the state type declared above in data it might make sense to declare a union type and use it in both locations.